src/frontend/src/handler/mod.rs

```diff
@@ -204,16 +205,35 @@ pub async fn handle(
             returns,
             params,
         } => {
-            create_function::handle_create_function(
-                handler_args,
-                or_replace,
-                temporary,
-                name,
-                args,
-                returns,
-                params,
-            )
-            .await
+            if let Some(_) = params.using {
+                // User defined function with external source (e.g., language [ python / java ])
+                create_function::handle_create_function(
```

Comment:
I think we should put this branching into the `create_function` module. and `create_sql_function.rs` should also be merged to `create_function.rs` so that some common logic can be reused.

Reply:
I create the new `create_sql_function.rs` to increase maintainability, especially the two kinds of udfs are not fully compatible, some detailed logic has subtle but critical difference. Merge these together may let the module hard to maintain and distinguish between different udf creating logic in the future.